etc/config/slang.amazon.properties

```diff
@@ -0,0 +1,14 @@
+compilers=&defaultslangc
+
+defaultCompiler=slangc
+supportsBinary=false
+compilerType=slang
+instructionSet=spirv
+
+disassemblerPath=/opt/compiler-explorer/SPIRV-Tools-master/build/tools/spirv-dis
+
+group.defaultslangc.compilers=slangc
```

Comment:
Let's not call this `slangc` -- that is too generic. This is `slangc_2024_14_6` or similar, then as we change and add new versions we don't invalidate old URLs that refer to `slangc`

Confirmed the path is OK:
```
$ /opt/compiler-explorer/slang-2024.14.6/bin/slangc -version
2024.14.6⏎                                    
```